packages/utilities/psammead-locales/moment/index.stories.jsx

```diff
@@ -46,10 +48,11 @@ const locales = [
   { name: 'Serbian', locale: 'sr' },
   { name: 'Serbian Cyrillic', locale: 'sr-cyrl' },
   { name: 'Tamil', locale: 'ta' },
+  { name: 'Gahuza', locale: 'ga' },
```

Comment:
```suggestion
  { name: 'Gahuza', locale: 'rw' },
```

Reply:
You have been faster than me noticing it :P